### PR TITLE
CNV_3044 Add Disk to VM

### PR DESCRIPTION
**Pull request by bmcelvee**
CP from https://github.com/openshift/openshift-docs/pull/17824. @bgaydosrh - FYI

### CODE REVIEW

**Comment by bmcelvee** [comment]
I'm not sure where the error is with this one.

**Comment by bmcelvee** [comment]
@vikram-redhat - this cherrypick is giving us a bit of trouble. Would you mind helping us troubleshoot?

File: modules/cnv-add-disk-to-vm.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+// and VM templates.
+
+// ifeval::["{context}" == "cnv-edit-vms"]
+// :cnv-vm-add-disk:
+// :object: virtual machine
+// :object-gui: Virtual Machines
+// endif::[]
+
+ifeval::["{context}" == "cnv-editing-vm-template"]
+:cnv-vm-add-disk-template:
```
Comment (aburdenthehand) [inline modules/cnv-add-disk-to-vm.adoc]: This ifdef is declared but not used (I'm pretty sure asciibinder dislikes that). We can remove this line.

File: modules/cnv-add-disk-to-vm.adoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+// :object!:
+// :object-gui!:
+
+// ifeval::["{context}" == "cnv-edit-vms"]
+// :cnv-vm-add-disk!:
+// endif::[]
+
+ifeval::["{context}" == "cnv-editing-vm-template"]
+:cnv-vm-template!:
```
Comment (aburdenthehand) [inline modules/cnv-add-disk-to-vm.adoc]: We can get rid of this too.
It also doesn't match what was declared earlier (and we're getting rid of that anyway)

File: modules/cnv-add-disk-to-vm.adoc (hunk 1 above)
Comment (bmcelvee) [reply modules/cnv-add-disk-to-vm.adoc]: Thanks, @aburdenthehand! I'll give it a go for both of your suggestions :)

**Comment by bmcelvee** [comment]
Andrew dug into this one a bit more and it sounds like the best path forward is to close this PR and make the changes on a PR specifically for 4.2. Thanks so much for looking into it!

**Pull request closed without merging**